public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/logs_view_config_panel_item.tsx

```diff
@@ -6,11 +6,12 @@
 import React, { useEffect, useState, useContext, Fragment } from 'react';
 import {
   EuiTitle,
-  EuiComboBox,
+  EuiCompressedComboBox,
   EuiSpacer,
+  EuiSmallButton,
```

Comment:
might have missed a button replacement here?

Reply:
There is an `EuiButton` with a `size` on this page as well.